src/fetch_foursquare_info.py

```diff
@@ -0,0 +1,129 @@
+import os.path
+import requests
+import numpy as np
+import pandas as pd
+import re
+import json
+import datetime
+
+# Setting dataset paths
+REIMBURSEMENTS_DATASET_PATH = os.path.join('data', '2016-11-22-reimbursements.xz')
+COMPANIES_DATASET_PATH = os.path.join('data', '2016-09-03-companies.xz')
+FOURSQUARE_DATASET_PATH = os.path.join('data', 'foursquare-companies.json')
+
+# These are the keys to access Foursquare's API. You can create your own through the link bellow
+# https://pt.foursquare.com/developers/register
+CLIENT_ID = ''
```

Comment:
I'd suggest using `config.ini` as in [`src/fetch_datasets.py`](https://github.com/datasciencebr/serenata-de-amor/blob/master/src/fetch_datasets.py#L23-L26) to keep API keys safe and restrict to environment variables (not to the repo).